### PR TITLE
Adding notebook to calculate and view thickness map

### PR DESCRIPTION
**Pull request by PranjalSahu**

### CODE REVIEW

File: notebooks/ComputeThicknessDemo.ipynb [hunk 1]
```diff
@@ -0,0 +1,1587 @@
...
+      "cell_type": "code",
+      "source": [
+        "# Main Code \n",
+        "\n",
+        "# Read the Segmentation Probability maps\n",
+        "fc_prob_file = 'FC_probmap.nii.gz'\n",
+        "tc_prob_file = 'TC_probmap.nii.gz'\n",
+        "\n",
+        "segmentation_file =  (fc_prob_file, tc_prob_file)\n",
+        "segmentation = [sitk.ReadImage(file) for file in segmentation_file]\n",
```
Comment (thewtex) [inline notebooks/ComputeThicknessDemo.ipynb:1265]: [itk.imread(file) for file in segmentation_file]

File: notebooks/ComputeThicknessDemo.ipynb [hunk 2]
```diff
@@ -0,0 +1,1587 @@
...
+        "\n",
+        "FC_prob = np.swapaxes(sitk.GetArrayFromImage(segmentation[0]), 0, 2).astype(float)\n",
```
Comment (thewtex) [inline notebooks/ComputeThicknessDemo.ipynb:1267]: np.asarray(segmentation[0])

**Review by thewtex — CHANGES_REQUESTED** [review]

**Comment by HastingsGreer** [comment]
The safest way to convert from indices to physical space is to use FC_prob_img.TransformIndexToPhysicalPoint( ... ) . This will handle all spacing, orientation, origin, etc.

**Comment by PranjalSahu** [comment]
Vectorizing some calculation to speed it up.

**Pull request merged**